ivy-posframe.el

```diff
@@ -268,7 +268,9 @@ This variable is useful for `ivy-posframe-read-action' .")
         (remove-text-properties 0 (length prompt) '(read-only nil) prompt)
         (with-current-buffer ivy-posframe-buffer
           (goto-char (point-min))
-          (kill-line 1)
+          (delete-region
+           (point)
+           (save-excursion (move-end-of-line 1) (point)))
```

Comment:
```
(delete-region (point) (save-excursion (line-move 1 'noerror) (point)))
```
This line is better. Your code remain '\n', ivy-posframe has unnecessary empty line
- your PR ver
> ![Capture 2019-06-01 20 35 22](https://user-images.githubusercontent.com/4703128/58747934-d2f4a300-84ac-11e9-93aa-2625e579ae76.png)

- Before `ivy-posframe--add-prompt` merge ver
> ![Capture 2019-06-01 20 36 44](https://user-images.githubusercontent.com/4703128/58747949-01727e00-84ad-11e9-883a-1b68c97e1f67.png)

But @tumashu may have another idea.

Those arguments are very short, It would be no problem to write in one line.

Reply:
Thanks for your opinion @conao3 
Another idea is simply adding `(setq kill-ring (cdr kill-ring))` after `(kill-line 1)`
How do you think of this?

Reply:
I think `delete-region` is better because a user may have another package related to kill-ring.

Reply:
Okay. I will update your suggestion.